dev/README_RELEASE_AIRFLOW.md

```diff
@@ -585,16 +585,23 @@ Airflow supports reproducible builds, which means that the packages prepared fro
 produce binary identical packages in reproducible way. You should check if the packages can be
 binary-reproduced when built from the sources.
 
-Checkout airflow sources and build packages in dist folder:
+Checkout airflow sources and build packages in dist folder (replace X.Y.Zrc1 with the version
+you are checking):
 
 ```shell script
-git checkout X.Y.Zrc1
+VERSION=X.Y.Zrc1
```

Comment:
`export` not needed?

Reply:
Not when you are using is only as parameter. I am merely using it so that I do not have to repeat it twice.

Export is only needed when you want to pass the variable version to the environment of the process you are starting as sub-process - but not when you are passing it as parameter for expansion in the same shell when your variable is defined (which is the case here as we are passing `--version ${VERSION}".

But yes, that reminds me that we have not defined it as an envariable `envvar="VERSION"`- which I just added. In this case if we exported it, we could not pass the `--version` parameter, at all (this is where `export` would be neceeassary). Still I think keepinvg `--version` as explicit flag is better in this case.